cassandra/pom.xml

```diff
@@ -10,16 +10,6 @@
 	<artifactId>cassandra</artifactId>
 	<dependencies>
 		<dependency>
-			<groupId>org.apache.cassandra</groupId>
```

Comment:
Cassandra and Common seems like a thing of same feather. The dependencies have to be sorted. I am aware of this.

You will find the below dependencies in common as well. Perhaps it is good for them to be in the same project.